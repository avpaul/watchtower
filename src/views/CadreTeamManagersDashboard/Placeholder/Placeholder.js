import React from 'react';
import PropTypes from 'prop-types';
import CustomLoader from '../../../components/CustomLoader/LoaderComponent';
import MoreImg from '../../../static/construction_new_color_loop.svg';
import '../Placeholders.scss';

const Placeholder = ({ text }) => (
  <div className="placeholder">
    <CustomLoader img={MoreImg} />
    <div className="font-weight-lighter">
      <h4 className="font-display"> {text}</h4>
    </div>
  </div>
);

Placeholder.propTypes = {
  text: PropTypes.func.isRequired
};

export default Placeholder;
