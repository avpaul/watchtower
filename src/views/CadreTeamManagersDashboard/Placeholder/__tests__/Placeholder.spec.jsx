import React from 'react';
import { shallow } from 'enzyme';
import Placeholder from '../Placeholder';

describe('Application component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Placeholder />);
  });

  it('renders placeholder component', () => {
    wrapper = shallow(<Placeholder />);
    expect(wrapper).toMatchSnapshot();
  });
});
