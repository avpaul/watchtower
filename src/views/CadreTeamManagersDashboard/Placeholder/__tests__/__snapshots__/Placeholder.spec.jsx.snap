// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application component renders placeholder component 1`] = `
<div
  className="placeholder"
>
  <LoaderComponent
    img="construction_new_color_loop.svg"
  />
  <div
    className="font-weight-lighter"
  >
    <h4
      className="font-display"
    >
       
    </h4>
  </div>
</div>
`;
